migrations/versions/05486ce795d5_make_session_name_required_on_messages.py

```
@@ -106,27 +106,49 @@ def upgrade() -> None:
+                            SELECT id
+                            FROM {schema}.messages
+                            WHERE workspace_name = '{workspace_name}'
+                            AND peer_name = '{peer_name}'
+                            AND session_name IS NULL
+                            LIMIT {batch_size}
+                        )
+                        UPDATE {schema}.messages
+                        SET session_name = '{default_session_name}'
+                        FROM batch
+                        WHERE messages.id = batch.id
+                    """)
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)